javalib/src/main/scala/java/util/NullRejectingHashMap.scala

```diff
@@ -80,6 +89,37 @@ private[util] class NullRejectingHashMap[K, V](
     super.remove(key)
   }
 
+  override def remove(key: Any, value: Any): Boolean = {
+    val old = this.get(key) // throws if `key` is null
```

Comment:
We usually do not qualify `this` if not necessary.

Reply:
Removed `this.` qualifier https://github.com/scala-js/scala-js/pull/4367/commits/9e530f3c125e9a1af2d9f12e238a1f5eea3f5f55